src/palace/manager/celery/tasks/identifiers.py

```diff
@@ -82,6 +86,18 @@ def mark_identifiers_unavailable(
     existing_identifier_kwargs, active_identifier_kwargs = (
         existing_and_active_identifier_sets
     )
+
+    if existing_identifier_kwargs is None or active_identifier_kwargs is None:
+        # If one of the sets is None, one of the tasks in the chord failed.
+        task.log.warning(
+            "Received None instead of IdentifierSet. Aborting without marking any identifiers as unavailable."
+        )
```

Comment:
Question: Is it useful to know which of the tasks aborted? Or will there already be enough information logged elsewhere to satisfy any need?

Reply:
I think it'll be notable that the calling task wants to abort this operation, so I expect it will log some detailed information about what went wrong, so I don't think that we need more detail here.